dlk/python/dlk/templates/src/matrix/generic/quantized_multiplication.cpp

```diff
@@ -31,7 +31,8 @@ static int pop_count(T_UINT i) {
   return (((i + (i >> 4)) & 0x0F0F0F0F) * 0x01010101) >> 24;
 }
 
-#define CONV(i, k) r##i##k += pop_count(~(a ^ b##k##0)) + 2 * pop_count(~(a ^ b##k##1)) - 3 * (pop_count(~a));
+#define CONV(i, k) r##i##k += pop_count(~(a ^ static_cast<QUANTIZED_PACKED::T>(b##k##0))) \
+                              + 2 * pop_count(~(a ^ static_cast<QUANTIZED_PACKED::T>(b##k##1))) - 3 * (pop_count(~a));
```

Comment:
How about making `pop_count()` and `operator~()` to be a member of `QuantizedPacked` or just creating standalone function accepting it.

e.g. (member version)
```
int PopCount() const { return pop_count(val); }
QuantizedPacked<packed_type> operator~() const { return QuantizedPacked<packed_type>(~val); }
```

allows us to simplify code above to
```
#define CONV(i, k) r##i##k += (~(a ^ b##k##0)).PopCount() + 2 * (~(a ^ b##k##1)).PopCount() - 3 * (~a).PopCount()
```

Reply:
Defined `pop_count` and `operator~`.